### PR TITLE
Updated Readme

### PR DESCRIPTION
**Pull request by MadMax1960**

### CODE REVIEW

**Comment by Tekisasu-JohnK** [comment]
Eeesh

**Review by RadiantUwU — APPROVED** [review]

**Review by Timo654 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,32 +1,32 @@
-# Godot Engine
+# Wokedot Engine
```
Comment (dandykong) [inline README.md:1]: Good lord, I hope this review system doesn't have automerge.

**Review by dandykong — CHANGES_REQUESTED** [review]

**Review by RadenTheFolf — COMMENTED** [review]
These changes have been addressed through pull request: https://github.com/Redot-Engine/redot-engine/pull/627

As such this pr can be closed.

**Comment by RadenTheFolf** [comment]
These changes have been addressed through pull request: https://github.com/Redot-Engine/redot-engine/pull/627

**Pull request closed without merging**